packages/schematics/angular/pwa/files/assets/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Also, I don't see the `manifest.json` included in `index.html` via `<link rel="manifest">`, is this normal ?